buildkite/src/Command/MinaArtifact.dhall

```diff
@@ -49,6 +53,7 @@ let MinaBuildSpec =
           , tags : List PipelineTag.Type
           , channel : DebianChannel.Type
           , debianRepo : DebianRepo.Type
+          , if : Optional B/If
```

Comment:
`if`s in this file were introduced but seem like remained unused. Maybe it's worth to revert this change?